02-IDE_exercises/solutions/06b-project/add_surname.py

```diff
@@ -1,9 +1,22 @@
-# Author: Brandon Hoffman
-# Date: 10/25/2020
-# Description: create a function that takes a list of names and stores any value that starts with the letter 'K' in a new list concatenated also with the string ' Hoffman'
+# Author: Brandon Hoffman Date: 10/25/2020 Description: create a function that takes a list of names and stores any
+# value that starts with the letter 'K' in a new list concatenated also with the string ' Hoffman'
 
-def add_surname(name_list):
+def add_surname(name_list: list[str]) -> list[str]:
```

Comment:
I don't think this will work. Should return an error? Need `from typing import List` and `List[str]` not `list[str]`